src/compiler/statement/SwitchStatement.ts

```diff
@@ -0,0 +1,41 @@
+import * as ts from "typescript";
+import {removeStatementedNodeChild} from "./../../manipulation";
+import {Expression, Node} from "./../common";
+import {ChildOrderableNode} from "./../base";
+import {Statement} from "./Statement";
+import {CaseBlock} from "./CaseBlock";
+
+export const SwitchStatementBase = ChildOrderableNode(Statement);
+export class SwitchStatement extends SwitchStatementBase<ts.SwitchStatement> {
+    /**
+     * Gets this switch statement's expression.
+     */
+    getExpression() {
+        return this.getNodeFromCompilerNode(this.compilerNode.expression) as Expression;
+    }
+
+    /**
+     * Gets this switch statement's case block.
+     */
+    getCaseBlock() {
+        return this.getNodeFromCompilerNode(this.compilerNode.caseBlock) as CaseBlock;
+    }
+
+    /**
+     * Get if the switch statement is exhaustive. False means it may or may not be exhaustive. See isNotExhaustive.
+     */
+    isExhaustive() {
+        return this.compilerNode.possiblyExhaustive === true;
```

Comment:
There must be a reason the underlying node says `possiblyExhaustive`. Maybe we should call this property `isPossiblyExhaustive()`?

Also, don't worry about also having the `isNotExhaustive` method. I believe `undefined` means `false` in this case.

Reply:
I interpreted it as `undefined` meant that the compiler/type checker doesn't know. Whereas `true` means definitely exhaustive and `false` means definitely not exhaustive. Thoughts?

Reply:
Oh ok, I just took a look at the code and this property is only set when it's bound by the type checker.

I'm going to remove both of these after merging.